playbooks/roles/ims_common/tasks/main.yml

```diff
@@ -71,7 +71,8 @@
           - active_ctl_msg is defined 
           - '"EMERGENCY BACKUP OR OVERRIDE" in active_ctl_msg | join'
 
-      - wait_for:
+      - name: Wait 10s for previous IMS command to complete
```

Comment:
Seeing this same task on other places, should we standardize it by adding the same name to all other occurrences?
Removing the name for all tasks also work.